README.md

```diff
@@ -97,11 +97,11 @@ with  @org.virtuslab.ash.annotation.SerializabilityTrait.
 
 
 The compiler plugin only checks the classes in the sbt modules where `AkkaSerializationHelperPlugin` is explicitly enabled.
-It may happen that your base trait (like `MySerializable` in the example) may live in an sbt module like `core` where you do **not** want to enable the plugin
-(e.g. for compilation performance reasons).
-You still need to reference `org.virtuslab.ash.SerializabilityTrait` to annotate `MySerializable`, however.
+It may happen that the base trait (like `MySerializable` in the example) lives in a sbt module like `core` where the plugin
```

Comment:
```suggestion
It may happen that the base trait (like `MySerializable` in the example) lives in an sbt module like `core` where the plugin
```
since it's `es-bee-tee` in pronunciation